geer26/panic_handling/docker-compose.yml

```
@@ -0,0 +1,20 @@
+#version: "3.9"
+services:
+
+
+#< ----------- API ------------ >
+
+  api:
+
+    container_name: api
```

```suggestion
       image: golang:1.22
       hostname: api
       container_name: api
       command: bash -c "go get ./... && go run ./cmd/backend1"
       working_dir: ./
```
you can also try "dev" approach, where you don't build prod image. (non go related)

---

Thank you for the suggestion, I'll look for documents and examples. Actually I currently have a project running on RPi3B, witch builds the modified source veeeryyyy slowly, This suggestion'd save me 8-12 minutes per build... For now I am leaving this in this way.